docs/getting-started/dockstore-tools.rst

```diff
@@ -205,193 +288,6 @@ information to be useful to other users. Selecting ``Hidden`` will
 prevent the tag from appearing in the public listing of tags for the
 image.
 
-CLI Client
```

Comment:
No, it's not being removed but I wasn't sure what to do with this. I will put this back in, but try to make it more clear that the `tool` keyword is only for "traditional" tools.  My original thinking for this and in other areas is to not acknowledge it since we want to eventually end github app tool registration. But maybe that's far off....

Reply:
Just to be clear: Is the goal to end *non*github app tool registration, or end github app tool registration? Is the plan to stop supporting gitlab etc?

Reply:
oops typo on my end. I've been under the assumption that we eventually want to end NON github app tool registration. @denis-yuen

Reply:
Checking with the "grand strategy" doc in the team drive. 
Yes, I believe the consensus was to deprecate and remove bitbucket and gitlab tools, then discourage manual tool registration in favor of github apps. In stages, this would end up simplifying things and give us less to maintain.

Reply:
That certainly makes sense, but if we're not there yet, I don't think it should be removed from our docs. Moved to another location, though, I'm all in favor of.